src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -226,71 +246,81 @@ export const SynapseNavDrawer: React.FunctionComponent<
           </a>
         </div>
         <List>
-          {currentUserProfile && (
+          {isLoggedIn && currentUserProfile && (
             <>
-              {getListItem({
-                tooltip: 'Projects',
-                iconName: 'dashboard',
-                onClickOpenNavMenu: NavItem.PROJECTS,
-              })}
-              {getListItem({
-                tooltip: 'Favorites',
-                iconName: 'favTwoTone',
-                onClickGoToUrl: `/#!Profile:${currentUserProfile.ownerId}/favorites`,
-              })}
-              {getListItem({
-                tooltip: 'Teams',
-                iconName: 'peopleTwoTone',
-                onClickGoToUrl: `/#!Profile:${currentUserProfile.ownerId}/teams`,
-              })}
-              {getListItem({
-                tooltip: 'Challenges',
-                iconName: 'challengesTwoTone',
-                onClickGoToUrl: `/#!Profile:${currentUserProfile.ownerId}/challenges`,
-              })}
-              {downloadListItem}
-              {isInSynapseExperimentalMode() &&
-                getListItem({
-                  tooltip: 'Trash Can',
-                  iconName: 'delete',
-                  onClickGoToUrl: '/#!Trash:0',
-                })}
+              <NavDrawerListItem
+                tooltip="Projects"
+                iconName="dashboard"
+                onClickOpenNavMenu={NavItem.PROJECTS}
+              />
+              <NavDrawerListItem
+                tooltip="Favorites"
+                iconName="favTwoTone"
+                onClickGoToUrl={`/#!Profile:${currentUserProfile.ownerId}/favorites`}
+              />
+              <NavDrawerListItem
+                tooltip="Teams"
+                iconName="peopleTwoTone"
+                onClickGoToUrl={`/#!Profile:${currentUserProfile.ownerId}/teams`}
```

Comment:
Followup ticket could be to add the number of team invitations + team requests as the badge content for the Teams item in the Nav